iris_grib/_load_convert.py

```diff
@@ -2076,6 +2076,37 @@ def product_definition_template_1(section, metadata, frt_coord):
     metadata['aux_coords_and_dims'].append((realization, None))
 
 
+def product_definition_template_6(section, metadata, frt_coord):
```

Comment:
This needs to be referenced in `product_definition_section()`:

https://github.com/SciTools/iris-grib/blob/101feb6478bd3e8eb28f79a1b4b1a844153074ef/iris_grib/_load_convert.py#L2476-L2514

Reply:
Hmm I think that might have got lost in manual merge ...